optaplanner-docs/src/main/asciidoc/ShadowVariable/ShadowVariable-chapter.adoc

```diff
@@ -44,7 +44,7 @@ So if A references B, then B references A.
 image::ShadowVariable/bidirectionalVariable.png[align="center"]
 
 For a non-chained planning variable, the bi-directional relationship must be a many to one relationship.
-To map a bi-directional relationship between two planning variables, annotate the master side (which is the genuine side) as a normal planning variable:
+To map a bi-directional relationship between two planning variables, annotate the source side (which is the genuine side) as a normal planning variable:
```

Comment:
Dtto.

Reply:
The same here - this is the wording we already have on the `main` branch.